app/classes/Transvision/ShowResults.php

```diff
@@ -479,7 +488,14 @@ public static function resultsTable($search_object, $search_results, $page)
                     <div dir='ltr' class='result_meta_link'>
                       <a class='source_link' href='{$locale2_path}'>
                         &lt;source&gt;
-                      </a>
+                      </a>";
+            if ($edit_link) {
```

Comment:
You need to set $edit_link to a null string, otherwise there will be a ton of warnings for locales not working in tools.